### DISCUSSION TITLE
command not found

### DISCUSSION THREAD

**Discussion by 0xcrow13** [projectdiscovery/subfinder#986] [Q&A]
sudo subfinder -h
sudo: subfinder: command not found
i keep getting command not found with most go tools even though i installed it the tool and golang

**Comment by MetzinAround** [comment]
This could be a PATH issue. This video talked about ensuring your PATH has the right directories in it. 

https://youtu.be/wtHcXA2l9c4?t=104

**Comment by 0xcrow13** [comment]
when i do `echo $PATH `
i get 
/home/kuro/.cargo/bin:/home/kuro/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/kuro/local/go/bin:/kuro/local/go/bin:/usr/local/go/bin: /home/kuro/go/bin

which is not the same as him even though i just uninstalled go 

and in my **.zshrc**  , my path is written like that 
export PATH="$PATH: /home/kuro/go/bin"

**Comment by hhthacker** [comment]
@kuromaku001 there should not be a space in your PATH string `export PATH="$PATH:/home/kuro/go/bin"`

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
